test/QuarkBuilder.t.sol

```diff
@@ -114,6 +118,69 @@ contract QuarkBuilderTest is Test {
         assertEq(result.multiQuarkOperationDigest, hex"", "empty multi digest");
     }
 
+    function testSimpleLocalTransferWithPaycallWrapperSucceeds() public {
```

Comment:
Would be good to test that the bridge operation is wrapped around a paycall successfully.